geomesa-fs/geomesa-fs-datastore/src/main/scala/org/locationtech/geomesa/fs/FileSystemFeatureIterator.scala

```diff
@@ -41,21 +40,21 @@ class FileSystemFeatureIterator(fs: FileSystem,
   override def close(): Unit = iter.close()
 }
 
-object CloseableEmptyIterator extends java.util.Iterator[SimpleFeature] with AutoCloseable  {
+object JavaCloseableIterator extends java.util.Iterator[SimpleFeature] with AutoCloseable  {
```

Comment:
I don't think you need this object at all.  You can call to CloseableIterator.empty[SimpleFeature], no?